bootstrap.env

```diff
@@ -0,0 +1,30 @@
+export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
+export CONJUR_MINOR_VERSION=5.0
+export CONJUR_APPLIANCE_IMAGE=registry.tld/conjur-appliance:$CONJUR_MINOR_VERSION-stable
+export CONJUR_FOLLOWER_COUNT=1
+export CONJUR_ACCOUNT=account-$UNIQUE_TEST_ID
+export AUTHENTICATOR_ID=conjur-$UNIQUE_TEST_ID-test
+export CONJUR_ADMIN_PASSWORD=ADmin123!!!!
+export DEPLOY_MASTER_CLUSTER=true
+export CONJUR_NAMESPACE_NAME=conjur-$UNIQUE_TEST_ID
+export TEST_RUNNER_IMAGE=demo-$UNIQUE_TEST_ID
+export DEV=false
+export APP_NAMESPACE_NAME=app-$UNIQUE_TEST_ID
+
+#######
+# Local DEV Env (uncomment all lines if using this configuration)
+#######
+# export PLATFORM=kubernetes
+# export AUTHENTICATOR_ID=authn-dev-env
+# export APP_NAMESPACE_NAME=local-secrets-provider
+# export CONJUR_NAMESPACE_NAME=local-conjur
+# export DEV=true
+# export STOP_RUNNING_ENV=true
+# export CONJUR_ACCOUNT=cucumber
+
+#######
+# Test App
```

Comment:
You updated all tests to App instead of Test App. You want to change here as well?

Reply:
I don't mind (: Updating

Reply:
@sigalsax when will users uncomment this? in which scenario?

Reply:
Deleting this scenario